tmf634/server/src/server.rs

```diff
@@ -600,6 +604,11 @@ impl<C> Api<C> for Server<C> where C: Has<XSpanIdString> + Send + Sync
         context: &C) -> Result<CreateResourceSpecificationResponse, ApiError>
     {
         info!("create_resource_specification({:?}) - X-Span-ID: {:?}", resource_specification, context.get().0.clone());
+        let mut con = self.redis_connection.clone();
+        match con.set("foo", 42i32).await {
```

Comment:
I can be wrong - but this hardcode should be not be there. We need find way to add tests and use specific test case for cover logic in tests. Otherwise it will hard to find the place with hardcode.